### PR TITLE
Debian packaging

### PR DESCRIPTION
**Pull request by kmansoft**
Add scripts and files for packaging into a .deb file

The scripts rely on package/VAR_VERSION and package/VAR_BUILD for version and build numbers. Please make sure those are up to date before cutting packages.

New make targets:

- victoria-metrics-package-deb: victoria-metrics

Builds and packages the amd64 version

- victoria-metrics-package-deb-arm64

Builds and packages the arm64 version

- victoria-metrics-package-deb-all

Both of the above, amd64 and arm64.

### CODE REVIEW

File: package/deb_control [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Package: victoria-metrics
```
Comment (valyala) [inline package/deb_control]: I'd move deb-related files to `/package/deb` subdirectory.

Comment (kmansoft) [reply package/deb_control]: Done, .deb specific files are now under `package/deb`.

**Comment by valyala** [comment]
@kmansoft , thanks for the pull request!

**Comment by kmansoft** [comment]
Thanks @valyala 

Hopefully we hear from @patsevanton on RPM - I'd like to learn that too for my personal project.

**Comment by valyala** [comment]
>  I'd like to learn that too for my personal project.

You can look at https://github.com/patsevanton/victoriametrics-rpm for details.

**Comment by patsevanton** [comment]
Great!
Where you want build and hosting packages?

**Pull request merged**